pkg/polaris-reporting/helpers.go

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2020 Synopsys, Inc.
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ *  with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package polarisreporting
+
+// SetHelmValueInMap adds the finalValue into the valueMap at the location specified
+// by the keyList
+// keyList - an ordered list of keys that lead to the location in the valueMap to place the finalValue
+func SetHelmValueInMap(valueMap map[string]interface{}, keyList []string, finalValue interface{}) {
+	currMap := valueMap
```

Comment:
why we need this?

Reply:
I was doing to for "code readabilty" and clarity. But I renamed some variables to make it clear